assets/aml-benchmark/tests/pipelines/inference_postprocessor_pipeline.yaml

```diff
@@ -15,25 +15,22 @@ inputs:
     type: uri_file
     path: ../data/sample_ground_truths.jsonl
   ground_truth_column_name: final_answer
-  prediction_probabilities_dataset:
-    type: uri_file
-    path: ../data/sample_ground_truths.jsonl
   separator: '\n\n'
   regex_expr: '(\d)'
-  strip_prefix: '.'
-  strip_suffix: '##'
+  remove_prefixes: '###,>>>'
+  strip_characters: '\n\n'
+  extract_number: 'first'
   template: '{{prediction}}'
-  extract_value_at_index: 0
   script_path:
     type: uri_file
-    path: ../data/test_custom_postprocessor.py
-  encoder_config: '{"column_name":"label", "0":"NEUTRAL", "1":"ENTAILMENT", "2":"CONTRADICTION"}'
-  
+    path: ../data/sample_ground_truths.jsonl

```

Comment:
Just curious if this is what was intended since the expectation of script_path is to accept a python script and not a jsonl file.

Reply:
@iamrk04 It's just a placeholder. All the values of the component's fields get set in test_inference_postprocessor.py